assignments/this.js

```diff
@@ -1,26 +1,60 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+* 1. Window Binding
```

Comment:
no explanations here, make sure you are comfortable explaining these different bindings